doc/userguide/partials/options.rst

```diff
@@ -25,16 +25,23 @@
 
 .. option:: -r <path>
 
-   Run in pcap offline mode reading files from pcap file. If <path> specifies
-   a directory, all files in that directory will be processed in order of
-   modified time maintaining flow state between files.
+   Run in pcap offline mode (replay mode) reading files from pcap file. If
```

Comment:
Hmm reading files from file sounds wrong. Is it just reading data or rather traffic from pcap file(s)?